src/domain/auth/auth.repository.ts

```diff
@@ -9,12 +9,15 @@ import {
   JwtClaimsSchema,
   JwtPayloadWithClaims,
 } from '@/datasources/jwt/jwt-claims.entity';
+import { ISafeRepository } from '@/domain/safe/safe.repository.interface';
 
 @Injectable()
 export class AuthRepository implements IAuthRepository {
   constructor(
     @Inject(IJwtService)
     private readonly jwtService: IJwtService,
+    @Inject(ISafeRepository)
+    private readonly safeRepository: ISafeRepository,
```

Comment:
This is not needed.

Reply:
Removed in 72f78b8985d92a7279ab99aafc3de01e84ecfdd7.